projects/i18n/languages/french/addon-editor.ts

```diff
@@ -35,5 +35,20 @@ export const TUI_FRENCH_LANGUAGE_ADDON_EDITOR: LanguageEditor = {
         insertHorizontalRule: 'Insérer la règle horizontale',
         superscript: 'Exposant',
         subscript: 'Indice',
+        insertTable: 'Insert table',
```

Comment:
When we merge, we need to ping all contributors to translations so they will add new prompts